From: Kurt Roeckx <kurt@roeckx.be>
Date: Sun, 06 Sep 2015 16:04:11 +0200
Subject: Disable SSLv3 test in test suite

When testing SSLv3 the test program returns 0 for skip.  The test for weak DH
expects a failure, but gets success.

It should probably be changed to return something other than 0 for a skipped
test.

Index: openssl-1.0.2d/test/testssl
===================================================================
--- openssl-1.0.2d.orig/test/testssl
+++ openssl-1.0.2d/test/testssl
@@ -160,7 +160,7 @@ test_cipher() {
 }
 
 echo "Testing ciphersuites"
-for protocol in TLSv1.2 SSLv3; do
+for protocol in TLSv1.2; do
   echo "Testing ciphersuites for $protocol"
   for cipher in `../util/shlib_wrap.sh ../apps/openssl ciphers "RSA+$protocol" | tr ':' ' '`; do
     test_cipher $cipher $protocol
